cms/conftest.py

```diff
@@ -25,6 +27,13 @@ def pytest_configure(config):
     """
     if config.getoption('help'):
         return
+    if hasattr(config, 'workerinput'):
+        # Set the process name for pytest-xdist workers to something
+        # recognizable, like "py_xdist_gw0", for the benefit of tools like
+        # New Relic Infrastructure and top
+        old_name = getproctitle()
+        new_name = 'py_xdist_{}'.format(config.workerinput['workerid'])
```

Comment:
Could we prefix with "python" instead of "py" to ensure that https://github.com/edx-ops/edx-minos/pull/23 works?